tests/unit/amo/reducers/test_versions.js

```diff
@@ -444,14 +444,49 @@ describe(__filename, () => {
     });
 
     describe('LOAD_RECOMMENDATIONS', () => {
+      const versionForRecommendations = {
+        ...version,
+        license: undefined,
+        release_notes: undefined,
+      };
+
       it('loads versions', () => {
         const state = versionsReducer(
           undefined,
           loadRecommendations({
             addons: [
               {
                 ...fakeAddon,
-                current_version: version,
+                current_version: versionForRecommendations,
+              },
+            ],
+            guid: fakeAddon.guid,
+            outcome: OUTCOME_RECOMMENDED,
+          }),
+        );
+
+        expect(
+          getVersionById({
+            state,
+            id: versionId,
+          }),
+        ).toEqual(createInternalVersion(versionForRecommendations));
+      });
+
+      it('does not overwrite license and releaseNotes', () => {
+        const slug = 'some-slug';
+        let state = versionsReducer(
+          undefined,
+          loadVersions({ slug, versions: [version] }),
```

Comment:
I think this would be a better test if it explicitly defined `license` and `release_notes` rather than rely on implicit values.